src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -313,6 +315,31 @@ private void LogScriptIfGiven(string type, string scriptPath)
             buildProperties[ManifestFilePropertyKeys.CompressDestinationDir] =
                 _cliOptions.CompressDestinationDir.ToString().ToLower();
 
+            // Override the prebuild and postbuild commands if BuildConfigurationFIle exists
+            if (context.SourceRepo.FileExists("app.yaml"))
+            {
+                try
```

Comment:
please add a logging statement stating the prebuild and postbuild commands will be overwritten

Reply:
Done